microovn/ovn/environment.go

```diff
@@ -179,3 +179,17 @@ func createPaths() error {
 
 	return nil
 }
+
+// removePaths removes directories created by createPaths function. This effectively removes
+// any data created during MicroOVN runtime.
+func removePaths() []error {
```

Comment:
While I like the way you think to be a good citizen and clean up after ourself, I think this should be a conscious choice of the operator, and not something we do automatically.

Would not calling `snap remove --purge` accomplish the same thing for the operator?

Reply:
Even the regular `snap remove` will accomplish it. The `--purge` just prevents creation of a snapshot.

I decided to remove all data files to prevent unintended ovsdb cluster re-connection. I'm going to double-test this, but I'm pretty sure that if we don't delete the files on departure and someone manually restarts services with `snap start microovn.central`, they'd try to re-join the cluster. This would create inconsistent state in which member would show up in `ovn-nbctl cluster/status` but it would not be present in `microovn cluster list`.

I agree that perhaps we can be less drastic with the cleanup and we could definitely keep at least `logs/` around. But I would for sure wipe at least `ovn.env` file and certificates. Do you see any value in leaving database files around?

I'd definitely have to re-test workflow
* join member
* kick member
* re-join member

which works nicely right now.

Reply:
As per our discussion. Instead of fully removing the directory, we can move & timestamp data directory. I'll make the change.